src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -184,14 +203,15 @@ uiModules.get('apps/management')
 
   this.createIndexPattern = () => {
     const id = this.newIndexPattern.name;
-    const timeFieldName =
-      this.newIndexPattern.isTimeBased
-      ? this.newIndexPattern.timeField.name
-      : undefined;
+    let timeFieldName;
+    if ((this.newIndexPattern.timeField !== TIME_FILTER_FIELD_OPTIONS.NO_DATE_FIELD_SELECTED)
+      && (this.newIndexPattern.timeField !== TIME_FILTER_FIELD_OPTIONS.NO_DATE_FIELDS_IN_INDEX)) {
```

Comment:
Yeah, I considered something like that but I find it less readable. In particular, I found the `.includes` relationship to be inverted (in terms of readability).

I'd be fine with shortening this if we could express this in reverse, so something like (pseudocode):

`!this.newIndexPattern.timeField.isOneOf([ ... ])`

But I don't think there's any way to do that, either in JS itself or using a helper library.

So I'd prefer to leave it the way it-is for added readability.

If you strongly disagree, we can get a third person to break the tie.